docs/reference/migration/migrate_2_1.asciidoc

```diff
@@ -41,4 +41,15 @@ thought of as a delete operation followed by an index operation.
 ==== `indices.fielddata.cache.expire`
 
 The experimental feature `indices.fielddata.cache.expire` has been removed.
-For indices that have this setting configured, this config will be ignored.
\ No newline at end of file
+For indices that have this setting configured, this config will be ignored.
+
+=== More Like This
+
+The MoreLikeThis builders have been moved to their own org.elasticsearch.index.query.morelikethis
+package. MoreLikeThisQueryBuilder.Item has been moved to its own class.
+
+The MoreLikeThisQueryBuilder#ignoreLike methods have been deprecating in favor
+to using the unlike methods.
+
+MoreLikeThisBuilder#addItem has been deprecated in favor to using
+MoreLikeThisBuilder#addLikeItem.
```

Comment:
yes @alexksikes and I did talk about this... but when I said this I was more thinking in terms of breaking changes in the context of a important bug fix or improvement. 

Refactoring an entire api is a bit too big of a change between minor versions, so I would just deprecate it in this change and in a follow up change only remove the old api for 3.0.0